src/components/DoctorCard/PageInfo.js

```diff
@@ -78,8 +191,15 @@ const PageInfo = function PageInfo({ doctor }) {
             </Tooltip>
           </Stack>
         </Stack>
-      </div>
-      <div>
+        {message === '' ? (
+          <div />
+        ) : (
+          <Alert sx={{ marginTop: '1rem' }} severity="success">
+            {message}
+          </Alert>
+        )}
```

Comment:
Do we need additional `<div/>`? If not this will be enough

```
{message && <Alert>{message}</Alert>
```

Reply:
Thanks, fixed it.